synapse/cores/ram.py

```diff
@@ -175,6 +180,25 @@ def _getSizeByProp(self, prop, valu=None, mintime=None, maxtime=None):
 
         return len(rows)
 
+    def _getCoreType(self):
+        return 'ram'
+
+    def _getBlobValu(self, key):
+        ret = self._blob_store.get(key)
+        return ret
+
+    def _setBlobValu(self, key, valu):
+        mesg = 'Setting blob value in ram cortex [{}]. Not a persistent action.'.format(self.myfo[0])
+        self.log(logging.WARNING, mesg=mesg, key=key, valu=valu)
```

Comment:
lets plumb events for this into the savefile events ( used by ram for persistence ) and rawk out.